lib/helpers/profileActiveHelpers.js

```diff
@@ -0,0 +1,9 @@
+if (Meteor.isClient){
+	Template.ProfileActive.helpers({
+		// TODO: Select Accepted listings that are of the users own id or have the acceptedOffer of the users offer
+		meetup: function () {
+			var id = Meteor.userId();
+			return Listing.find({ creator_id: Meteor.userId(), status: "Accepted" });
```

Comment:
Line is too long.<br>Identifier 'creator_id' is not in camel case.<br>'Listing' is not defined.<br>'Meteor' is not defined.